be/src/common/config.h

```diff
@@ -516,10 +514,23 @@ CONF_Int32(flush_thread_num_per_store, "2");
 // config for tablet meta checkpoint
 CONF_mInt32(tablet_meta_checkpoint_min_new_rowsets_num, "10");
 CONF_mInt32(tablet_meta_checkpoint_min_interval_secs, "600");
+// Thread count of RocksDB uses for background flush and compaction, -1 means the number of cores.
+CONF_mInt32(rocksdb_thread_count, "-1");
+CONF_mInt32(rocksdb_block_cache_mb, "1024");
+CONF_Validator(rocksdb_block_cache_mb, [](int value) -> bool {
+  return value > 0;
+});
+CONF_String(rocksdb_compression_type, "LZ4");
```

Comment:
What is the difference between LZ4 and snappy? Is it necessary to config this?

Reply:
> Use options.compression to specify the compression to use. By default it is Snappy. We believe LZ4 is almost always better than Snappy. We leave Snappy as default to avoid unexpected compatibility problems to previous users.

https://github.com/facebook/rocksdb/wiki/Compression

We encountered a problem that snappy decompress cost too much time in product environment.
Of course, the root cause is that the rocksdb value is too large, some tablet meta which have thounds of rowsets may reach hundreds of MB. But I think we can improve it by use LZ4.